### DISCUSSION TITLE
Stuttering possibly introduced by latest build(s)

### DISCUSSION THREAD

**Discussion by kamild1996** [ppy/osu#25736] [Q&A] [answered]
Video attached. Frametime jumps can be seen throughout the whole play, they're susceptible but not as serious as some outliers - a few seconds after following timestamps: 0:17, 1:13, 2:11, 2:23
[lazor.webm](https://github.com/ppy/osu/assets/5320414/b047dbab-c84f-4060-8483-24ec34d4ce98)
I believe this may be a regression. I only play osu!lazer maybe once or twice a month just to check out what changed, thus I don't recall the previous build version I had, which did not display these issues. Logs:
[updater.log](https://github.com/ppy/osu/files/13653401/updater.log)
[database.log](https://github.com/ppy/osu/files/13653402/database.log)
[input.log](https://github.com/ppy/osu/files/13653403/input.log)
[network.log](https://github.com/ppy/osu/files/13653404/network.log)
[performance.log](https://github.com/ppy/osu/files/13653405/performance.log)
[runtime.log](https://github.com/ppy/osu/files/13653400/runtime.log)

**Comment by bdach** [comment]
please capture another video with frame graph shown (ctrl-f11 twice).

**Comment by peppy** [comment, marked as answer]
Fixed via many many PRs to reduce allocations. Still some further improvements coming.

**Comment by kamild1996** [comment]
Apologies for not responding in a timely manner. I tried to get those large spikes to happen again, but was unable to. I can confirm however that the current version of the game does feel noticeably smoother, spikes are small and so far apart I can't notice them without the graphs open.
https://streamable.com/wyph00
